xpk.py

```diff
@@ -4412,6 +4457,32 @@ def directory_path_type(value):
       ' approval.'
     ),
 )
+cluster_create_optional_arguments.add_argument(
+    '--create-vertex-tensorboard',
+    action='store_true',
+    help='Set this flag to create a Tensorboard instance in Vertex AI.',
+)
+cluster_create_optional_arguments.add_argument(
+    '--tensorboard-region',
+    type=str,
+    default='us-central1',
+    help=(
+      'The region to create Vertex Tensorboard instance in. '
+      'Visit https://cloud.google.com/vertex-ai/docs/general/locations#available-regions '
+      'to view regions supported by Vertex AI. By default, Tensorboard instance will '
+      'be created in us-central1.'
```

Comment:
Vertex AI Tensorboard is not available in all regions. If we set the default to cluster region, then there is a possibility that Vertex AI APIs won't work. For example, cluster is in `us-central2`, then we would see API error because Vertex AI is not available in that region. That's the reason I had to create another argument for this. wdyt?

Reply:
I see I see makes sense, this small piece is worth thinking about in the user journey for Vertex AI usage. Would most users want t o use us-central1? when would they want to pick a new region beside that?

Reply:
Typically, users should choose the Vertex location closest to their physical location. That's the reason we have this argument available to users to add. I can remove the default region if you think that would make more sense.